recipes/wrf-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0b1" %}
+
+package:
+  name: wrf-python
+  version: {{ version }}
+
+source:
+  fn: wrf-python-{{ version }}.tar.gz
+  url: https://github.com/NCAR/wrf-python/archive/{{ version }}.tar.gz
+  sha256: 94f05a9719701959702094eafb84555a996bc4f7d75042fb07bca3c35bda9f31
+
+build:
+  number: 1
+  detect_binary_files_with_prefix: true
+  skip: True  # [win32 or np>=112 or (win and py>=35)]
+
+requirements:
+  build:
+    - pip
+    - numpy x.x
+    - wrapt
+    - m2w64-toolchain  # [win]
+    - gcc  # [unix]
+    - libgcc  # [unix]
+    - python
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
+  run:
+    - numpy x.x
+    - python
+    - wrapt 
+    - m2w64-toolchain  # [win]
+    - gcc  # [unix]
+    - libgcc  # [unix]
```

Comment:
I wasn't sure about this one.  I think libgfortran is for OSX only, and libgcc is supposed to have the fortran symbols in them, and then one day that stopped working, so I just pulled the whole gcc in as well.  I'll play around with it some more.